legacy-standalone/src/main/java/com/google/android/apps/muzei/legacy/TLSSocketFactory.java

```diff
@@ -74,10 +74,10 @@ public Socket createSocket(InetAddress address, int port, InetAddress localAddre
         return enableTLSOnSocket(delegate.createSocket(address, port, localAddress, localPort));
     }
 
-    private Socket enableTLSOnSocket(Socket socket) {
-        if ((socket instanceof SSLSocket)) {
-            ((SSLSocket) socket).setEnabledProtocols(new String[]{"TLSv1.1", "TLSv1.2"});
-        }
-        return socket;
-    }
+    private static Socket enableTLSOnSocket(Socket socket) {
+		if ((socket instanceof SSLSocket)) {
```

Comment:
Remove this extra layer of indentation.

Reply:
Thank you for your feedback. Yeah, I already know this issue, it's because the tool works through ASTs, and I leave the default reformatting of the library when i write back the changes into source files.

Reply:
You'll need to restore the correct indentation before this can be merged in. As far as I see it, the only change should be the addition of the `static` word.